MediaGalleryUi/view/adminhtml/web/js/image/image-details.js

```diff
@@ -137,8 +138,8 @@ define([
 
             if (_.isObject(value)) {
                 $.each(value, function (moduleName, count) {
-                    usedIn += count + ' ' +
-                        this.getEntityNameWithPrefix(moduleName, count) +
+                    usedIn += '<a href="'+ this.getFilterUrl(moduleName) +'">' + count + ' ' +
```

Comment:
It would be good to move the HTML to the template and use foreach binding to render items